### PR TITLE
Use `onupdate` for `last_updated_at`

### LINKED ISSUE #1061: Use `onupdate` for `last_updated_at` fields
## What issue are you seeing?
Instead of manually updating the `last_updated_at` values, we should be using the `onupdate=func.now()` syntax.

### PR DESCRIPTION
**Pull request by michplunkett**
## Fixes issue
https://github.com/lucyparsons/OpenOversight/issues/1061

## Description of Changes
Instead of manually updating `last_updated_at`, we should let SQL Alchemy take care of it.

## Tests and linting
 - [x] This branch is up-to-date with the `develop` branch.
 - [x] `pytest` passes on my local development environment.
 - [x] `pre-commit` passes on my local development environment.

### CODE REVIEW

File: OpenOversight/app/main/model_view.py [hunk 1]
```diff
@@ -222,7 +221,6 @@ def populate_obj(self, form, obj):
             obj.created_by = current_user.id
         # if the object keeps track of who updated it last, set to current user
         if hasattr(obj, "last_updated_at"):
```
Comment (sea-kelp) [inline OpenOversight/app/main/model_view.py]: Can you change this to "last_updated_by"?

File: OpenOversight/app/models/database_imports.py [hunk 2]
```diff
@@ -338,12 +334,15 @@ def update_incident_from_dict(data: Dict[str, Any], incident: Incident) -> Incid
             )
         )
     if "last_updated_by" in data:
-        incident.last_updated_by = parse_int(data.get("last_updated_by"))
+        incident.last_updated_by = parse_int(
+            data.get(
+                "last_updated_by", User.query.filter_by(is_administrator=True).first()
```
Comment (sea-kelp) [inline OpenOversight/app/models/database_imports.py:339]: Would it be better to just leave it as null?

**Review by sea-kelp — COMMENTED** [review]
Looks good! A couple comments:

File: OpenOversight/app/models/database_imports.py (hunk 2 above)
Comment (michplunkett) [reply OpenOversight/app/models/database_imports.py:339]: We're doing this for the rest of the last_updated_by fields, so I'd at least prefer it be the same for all updates. We could modify it to be either empty or this some other time, imo.

File: OpenOversight/app/main/model_view.py (hunk 1 above)
Comment (michplunkett) [reply OpenOversight/app/main/model_view.py]: Done!

**Review by sea-kelp — APPROVED** [review]

**Pull request merged**